modules/nw-ovn-kubernetes-migration.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/migrate-from-openshift-sdn.adoc
+
+[id="nw-ovn-kubernetes-migration_{context}"]
+= Migrating to the OVN-Kubernetes network provider
+
+As a cluster administrator, you can upgrade the Pod network provider for your cluster to the OVN-Kubernetes Pod network provider.
+During the migration, you must reboot every node in your cluster.
+
+.Prerequisites
+
+* Install the OpenShift Command-line Interface (CLI), commonly known as `oc`.
+* Access to the cluster as a user with the `cluster-admin` role.
+* A cluster installed on bare metal infrastructure configured with the OpenShift SDN Pod network provider.
+
+.Procedure
+
+. To backup the configuration for the cluster network, enter the following command:
++
+----
+$ oc get Network.config.openshift.io cluster -o yaml
+----
+
+. To enable the migration, set an annotation on the Cluster Network Operator configuration object by entering the following command:
++
+----
+$ oc annotate Network.operator.openshift.io cluster \
+  'networkoperator.openshift.io/network-migration'=""
+----
+
+. To change the network provider, enter the following command:
++
+----
+$ oc patch Network.config.openshift.io cluster \
+  --type='merge' --patch '{ "spec": { "networkType": "OVNKubernetes" } }'
+----
+
+. To confirm the migration disabled the OpenShift SDN network provider, enter the following command. It might take several moments for all the OpenShift SDN Pods to stop.
++
+----
+$ oc get pod -n openshift-sdn --watch
+----
+
+. To complete the migration, reboot each node in your cluster. For example, the following bash script reboots each node in your cluster. The script assumes that you can connect to each host by using `ssh` and that you have configured `sudo` to not prompt for a password.
++
+[source,bash]
+----
+for ip in `oc get node -o wide | egrep -v "NAME" |awk '{print $6}'`
```

Comment:
how do we feel about jsonpath?

```bash
for ip in $(oc get nodes  -o jsonpath='{.items[*].status.addresses[?(@.type=="InternalIP")].address}')
```

otherwise we could avoid the `egrep -v` with `oc get nodes -o wide --no-headers=true`

Reply:
@rbbratta I'm a fan of all paths!